DuckDuckGo/TabViewController.swift

```diff
@@ -1007,7 +1007,7 @@ class TabViewController: UIViewController {
     }
 
     @IBAction func onBottomOfScreenTapped(_ sender: UITapGestureRecognizer) {
-        showBars(animated: false)
+        showBars()
```

Comment:
Not sure if this was on purpose, but tested with animation (the same as when tapping area on top) and it worked.